### ISSUE TITLE
annos['num_points_in_gt'] are all 0 in Waymo Dataset (multi-frame)

### ISSUE THREAD

**Issue by rex131** [open-mmlab/OpenLidarPerceptron#1707]
Hi, I am trying to train MPPNet. I have trained centerpoint_4frames. I found something wrong when I was laoding the dataset. Here is the detail:

 if self.training and len(data_dict['gt_boxes']) == 0:
            new_index = np.random.randint(self.__len__())
            return self.__getitem__(new_index)

The loader gets stuck in an endless loop of iterations because of data_dict['gt_boxes']==0. It seems annos['num_points_in_gt'] are all 0 in data infos.
This problem arise when I do not use gt sampler.

**Comment by Petros626** [comment]
Normally this part of the code replaces empty samples, which means for example a KITTI .bin file could contain a scene where the annotation is far away (outside the pc range) the label (gt_box) isn't valid then and gets replaced by another valid sample from train.txt. The waymo dataset should contain objects in every sample.

Whats your yaml configuration? Maybe the data isn't inserted properly in the pkl file.
